R/lat_bins_equal.R

```diff
@@ -0,0 +1,114 @@
+#' Generate equal-area latitudinal bins
+#'
+#' A function to generate approximately equal-area latitudinal bins for a
+#' user-specified given number of bins and latitudinal range. This approach is
+#' based on calculating the curved surface area of spherical segments bounded
+#' by two parallel disks.
+#'
+#' @param n \code{numeric}. A single numeric value defining the number of
+#'   equal-area latitudinal bins to split the latitudinal range by (as defined
+#'   by `min` and `max`).
+#' @param min \code{numeric}. A single numeric value defining the lower limit
+#'   of the latitudinal range (defaults to -90).
+#' @param max \code{numeric}. A single numeric value defining the upper limit
+#'   of the latitudinal range (defaults to 90).
+#' @param r \code{numeric}. The radius of the Earth in kilometres. Defaults to
+#'   the the mean radius of the Earth (6371 km).
```

Comment:
Do we need to expose `r`? I can't imagine a scenario where a user of our package would want to change this.

Reply:
I don't think users will change this either tbh, but when I went on a bit of a deep dive of this, there are a few different values that individuals use. I think by giving a common default, we're being transparent about it and allowing users to change this if they really want to.

Reply:
Sure, so in theory someone could use a value, for example, with more decimal places. In that case, can we add some sort of citation to the parameter description for this value and/or state clearly that this allows numbers from other sources to be used?